package.json

```diff
@@ -61,6 +61,7 @@
     }
   },
   "dependencies": {
+    "jotai": "2.14.0",
```

Comment:
introduced jotai finally to seamless state management. its a drop-in replacement for useState often. for example, useLocalStorage is not global state but only local. jotai binding makes it global when needed